README

```diff
@@ -5,9 +5,16 @@ Apache module mod_lookup_identity
 Apache module to lookup user identifier and retrieve additional
 information about the authenticated user.
 
-The module supports lookup of the user identifier based on certificate
-via SSSD D-Bus call org.freedesktop.sssd.infopipe.Users.FindByCertificate,
-setting up the r->user request structure.
+The module supports lookup of the user identifier based on
+certificate via SSSD D-Bus call, setting up the r->user request
+structure.
+Method org.freedesktop.sssd.infopipe.Users.FindByNameAndCertificate
+is used by default and the module also looks for configurable variable
+from query string and sends it to SSSD together with certificate to
```

Comment:
Instead of "module also looks for ..." I'd use something like "value of query string parameter configured with ... directive ... is passed to SSSD together ..."